acdc-ws/app/utils/DbConfig.scala

```diff
@@ -22,4 +23,11 @@ class DbConfig() {
     case Failure(e) => throw e
   }
 
+  def countTaskFrequencyMinute: Int = Try(config.getInt(s"count-task-frequency-minute")) match {
+    case Success(d) => d
+    case _ =>
```

Comment:
let's only match on not exist exception, and error on other

Reply:
got it.  changes pushed.